### PR TITLE
Set pkg_mgr facts manually for FreeBSD and OpenSUSE

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Should improve integration tests in CI.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

**Comment by felixfontein** [comment]
Works around ansible/ansible#67684 not (yet) being merged: it makes `package` work also on FreeBSD and OpenSuSE. This makes a lot more things pass CI.

I plan to revert this once it is no longer necessary (i.e. when that PR in ansible/ansible is merged).

File: tests/integration/targets/setup_pkg_mgr/tasks/main.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ---
 - set_fact:
     pkg_mgr: community.general.pkgng
+    ansible_pkg_mgr: community.general.pkgng
```
Comment (sshnaidm) [inline tests/integration/targets/setup_pkg_mgr/tasks/main.yml]: I wonder if it's possible to run setup module for `ansible_pgk_mgr` discovery like here: https://github.com/ansible/ansible/blob/308723c3ca1122363e419070f1fa1d76ff5611a9/lib/ansible/plugins/action/package.py#L54

Comment (felixfontein) [reply tests/integration/targets/setup_pkg_mgr/tasks/main.yml]: No (besides that that's already done). The problem is that `setup` returns the wrong name, which currently (until ansible/ansible#67684 is merged) does not work. The action plugin `package` will try to find the module `pkgng` (resp. `zypper`), but the modules are called `community.general.pkgng` (resp. `community.general.zypper`) now.

With the routing PR in ansible/ansible, the short name will again work (and this PR can be reverted). But until then, this is needed for make CI work for FreeBSD and OpenSuSE.

Comment (sshnaidm) [reply tests/integration/targets/setup_pkg_mgr/tasks/main.yml]: clear, thanks

**Comment by felixfontein** [comment]
Thanks for reviewing this one as well @sshnaidm!

**Pull request merged**